microsoft-edge/toc.yml

```
@@ -848,6 +848,10 @@
             - name: View extensions analytics dashboard
               href: extensions-chromium/publish/extensions-analytics.md
 
+            - name: Respond to user reviews
+              href: extensions-chromium/publish/reply-user-reviews.md
+              displayName: Reply to user reviews
```

```suggestion
            - name: Reply to user reviews
              href: extensions-chromium/publish/reply-user-reviews.md
              displayName: Respond to user reviews
```
commit this change if retitle the article